routes/web.php

```diff
@@ -12,22 +12,19 @@
   | kernel and includes session state, CSRF protection, and more.
   |
  */
-
-
 Route::group(['middleware' => ['web']], function () {
-    Route::get('taskList','TaskController@taskList');
-    Route::get('taskDetail/{id}','TaskController@taskDetail');
-    Route::post('task','TaskController@addNewTask');
-    Route::delete('task/{id}','TaskController@deleteTask');
-    Route::get('listUserCanAdd/{id}','TaskController@listUserCanAdd');
-    Route::get('addUserToTask/{id}', 'TaskController@addUserToTask');
-    Route::get('removeUserFromTask/{id}','TaskController@removeUserFromTask');
-    Route::get('userList','UserController@userlist');
-    Route::get('userDetail/{id}','userController@userDetail');
-    Route::post('user','UserController@addNewUser');
-    Route::delete('user/{id}','UserController@deleteUser');
-    Route::get('listTaskCanAdd/{id}','UserController@listTaskCanAdd');
-    Route::get('listTaskCanAdd/{id}','UserController@listTaskCanAdd');
-    Route::get('addTaskToUser/{id}', 'UserController@addTaskToUser');
-    Route::get('removeTaskFromUser/{id}','UserController@removeTaskFromUser');
-});
+    Route::get('taskList', 'TaskController@index')->name('taskList');
```

Comment:
xem lại cách đặt tên route
```suggestion
    Route::get('task-list', 'TaskController@index')->name('task_list');
```